src/main/java/io/fabric8/jenkins/openshiftsync/CredentialsUtils.java

```diff
@@ -282,8 +284,33 @@ private static Credentials secretToCredentials(Secret secret) {
                     data.get(OPENSHIFT_SECRETS_DATA_USERNAME),
                     data.get(OPENSHIFT_SECRETS_DATA_SSHPRIVATEKEY));
         default:
-            logger.log(Level.WARNING,
-                    "Unknown secret type: " + secret.getType());
+			String type = secret.getType();
```

Comment:
i think this indenting is wrong?  (this is part of the "default:" case right?

Reply:
yeah more fabric team formatting woes ... we reformat the entire file in a separate commit